cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -414,6 +383,33 @@ instance IsEffectuator (LiveViewBackend blk) Text where
 
                                 return $ lvs { lvsTransactions = fromIntegral txsProcessedNum }
 
+                    LogValue "blocksForgedNum" (PureI forgedBlocksNum) ->
+                        modifyMVar_ (getbe lvbe) $ \lvs -> do
+
+                                checkForUnexpectedThunks ["blocksForgedNum LiveViewBackend"] lvs
+
+                                return $ lvs { lvsBlocksMinted = fromIntegral forgedBlocksNum }
+                    LogValue "nodeIsLeaderNum" (PureI leaderNum) ->
+                        modifyMVar_ (getbe lvbe) $ \lvs -> do
+
+                                checkForUnexpectedThunks ["nodeIsLeaderNum LiveViewBackend"] lvs
+
+                                return $ lvs { lvsLeaderNum = fromIntegral leaderNum }
+
+                    LogValue "slotsMissedNum" (PureI missedSlotsNum) ->
+                        modifyMVar_ (getbe lvbe) $ \lvs -> do
+
+                                checkForUnexpectedThunks ["slotsMissedNum LiveViewBackend"] lvs
+
+                                return $ lvs { lvsSlotsMissedNum = fromIntegral missedSlotsNum }
+
+                    LogValue "forksCreatedNum" (PureI createdForksNum) ->
+                        modifyMVar_ (getbe lvbe) $ \lvs -> do
+
+                                checkForUnexpectedThunks ["forksCreatedNum LiveViewBackend"] lvs
+
+                                return $ lvs { lvsCreatedForksNum = fromIntegral createdForksNum }
+
```

Comment:
This chunk of 3 lines is repeated over and over again. Maybe it should be a function.